src/main/java/com/google/graphgeckos/dashboard/storage/GCDataRepository.java

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.graphgeckos.dashboard.storage;
+
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.cloud.datastore.StructuredQuery.PropertyFilter;
+import com.google.cloud.datastore.TimestampValue;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.Timestamp;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.Date;
+import java.util.List;
+import org.springframework.boot.SpringApplication;
+import org.springframework.cloud.gcp.data.datastore.core.DatastoreTemplate;
+import org.springframework.stereotype.Repository;
+
+/**
+ * Implementation of the DataRepository, with the purpose of providing I/O to the Google Cloud
+ * Datastore. Each individual database entry is modeled by the {@link #BuildInfo BuildInfo} class.
+ * The relevant fields for the database, which are described in that class are:
+ *    - Kind: "revision"
+ *    - Key: commit hash
+ * The database indexes after all the BuildInfo fields, besides the {@code builders} list.
```

Comment:
```suggestion
 * The database indexes all BuildInfo fields, except the {@code builders} list.
```
([besides vs. except](https://english.stackexchange.com/questions/7984/difference-between-except-and-besides#:~:text=They%20have%20totally%20different%20meanings,besides%20serves%20that%20of%20inclusion.&text=The%20whole%20class%20is%20going%2C%20except%20John.&text=Besides%20the%20class%2C%20Mary%20is%20also%20going.))

Reply:
removed the indexing part, according to the next comment